src/main/java/com/example/dockernasaproxy/controller/NasaController.java

```diff
@@ -0,0 +1,21 @@
+package com.example.dockernasaproxy.controller;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.example.dockernasaproxy.service.PictureService;
+
+import lombok.AllArgsConstructor;
+
+@RestController
+@AllArgsConstructor
+public class NasaController {
+
+    private final PictureService pictureService;
+
+    @GetMapping(value = "/mars/pictures/largest", produces = "image/jpeg")
```

Comment:
You can import the media type (`produces` value) as a constant from the `org.springframework.http.MediaType` to avoid hardcoding it.